Terminal.Gui/View/View.cs

```diff
@@ -1756,18 +1756,15 @@ protected void ClearNeedsDisplay ()
 		/// <returns></returns>
 		public virtual bool OnDrawFrames ()
 		{
-			if (!IsInitialized) {
-				return false;
-			}
+			//if (!IsInitialized) {
+			//	return false;
+			//}
 
 			var prevClip = Driver.Clip;
-			Driver.Clip = ViewToScreen (Frame);
 
-			// TODO: Figure out what we should do if we have no superview
-			//if (SuperView != null) {
-			// TODO: Clipping is disabled for now to ensure we see errors
-			Driver.Clip = new Rect (0, 0, Driver.Cols, Driver.Rows);// screenBounds;// SuperView.ClipToBounds ();
-										//}
+			if (SuperView != null) {
```

Comment:
Please remove this. See my comment above.

I plan on re-enabling clipping later; it's important to see where things are being drawn where they're not supposed to be before we re-implement a proper clipping mechanism (the one in v1 was badly architected and broken).